api/v1alpha1/modelregistry_types.go

```diff
@@ -173,6 +173,10 @@ type RestSpec struct {
 	// Optional image to support overriding the image deployed by the operator.
 	//+optional
 	Image string `json:"image,omitempty"`
+
+	// Optional catalog data image to support overriding the default catalog data source.
+	//+optional
+	CatalogDataImage string `json:"catalogDataImage,omitempty"`
```

Comment:
@pboyd and I discussed this and this variable doesn't really belong in the MR CRD. Since catalog service is created without any MR instances, the image env variable in params.env is the only mechanism to configure the model registry operator to use a particular catalog data image. 
```suggestion
```